assemble/pom.xml

```diff
@@ -171,6 +171,9 @@
             <excludes>
               <exclude>bin-LICENSE</exclude>
               <exclude>bin-NOTICE</exclude>
+              <exclude>nbproject/**</exclude>
+              <exclude>nb-configuration.xml</exclude>
+              <exclude>nbactions.xml</exclude>
```

Comment:
It is possible to run a maven project without generating any of these files.  But say I want to run Netbeans with Java 8 and compile the project with Java 7, then a nb-configuration.xml is created.  That file is use for other configs too.  If I want to run a custom task, like mvn clean test -Psunny, a nbactions.xml file is created.  The nbproject directory is created for any non maven projects.  I don't actually have any nbproject but a user could create one for say some of the native code or some of the python code in the test module.

I have been unable to get Netbeans to not require these files under each sub module.